Formula/stern.rb

```diff
@@ -0,0 +1,32 @@
+class Stern < Formula
+  desc "Tail multiple Kubernetes pods & their containers."
+  homepage "https://github.com/wercker/stern"
+  url "https://github.com/wercker/stern.git",
+    :tag => "1.0.1",
+    :revision => "3c8e6184b7a0a0cbc8dc78f219be0a49a007e861"
+
+  head "https://github.com/wercker/stern",
+    :shallow => false
+
+  depends_on "go" => :build
+  depends_on "govendor" => :build
+
+  def install
+    contents = Dir["{*,.git,.gitignore}"]
+    gopath = buildpath/"gopath"
+    (gopath/"src/github.com/wercker/stern").install contents
```

Comment:
Apparently not. I'm unable to pull dependencies via `govendor sync` if I symlink. Full disclosure: my golang-fu is not strong.